lib/bin/filter_clipped_alignments.py

```diff
@@ -50,6 +65,14 @@ def main(reads_to_contig_sam, output_file, max_percent):
                 percent_clipped = ((front_clipping + end_clipping) / alength) * 100
                 if percent_clipped < max_percent:
                     output.writerow(row)
```

Comment:
Out of scope for this PR, but it might be useful to write this script using the [pysam library](https://pysam.readthedocs.io/en/latest/api.html) so you don't have to deal with column numbers, cigar parsing, TSV parsing, etc. It also lets you open a sam/bam file for reading and another one for writing!

Reply:
Thanks @robertaboukhalil I realized this like 80% through haha